admin/query/pom.xml

```diff
@@ -79,6 +79,12 @@
                         <Embed-Dependency>
                             admin-query-common
                         </Embed-Dependency>
+                        <Import-Package>
+                          org.codice.ddf.admin.configurator;version="[2.23,3)",
+                          org.codice.ddf.configuration;version="[2.23,3)",
+                          org.codice.ddf.internal.admin.configurator.actions;version="[2.23,3)",
+                            *
```

Comment:
❓ Why the need to import ddf artifacts where we didn't before?

Reply:
This is what is letting us use 2.23.0 ddf dependencies instead of requiring the 2.25.2 ones we build with. This gives us the backwards compatibility needed to work with alliance 1.10.x